py/desisim/scripts/quickquasars.py

```diff
@@ -370,27 +369,31 @@ def simulate_one_healpix(ifilename,args,model,obsconditions,decam_and_wise_filte
                 for dla in dla_info[dla_info['MOCKID']==idd]:
 
                     # Adding only DLAs with z < zqso
-                    if dla['Z_DLA']>=metadata['Z'][ii]: continue
-                    dlas.append(dict(z=dla['Z_DLA']+dla['DZ_DLA'],N=dla['N_HI_DLA']))
+                    if dla['Z_DLA_RSD']>=metadata['Z'][ii]: continue
+                    dlas.append(dict(z=dla['Z_DLA_RSD'],N=dla['N_HI_DLA'],dlaid=dla['DLAID']))
                 transmission_dla = dla_spec(trans_wave,dlas)
 
             elif args.dla=='random':
                 dlas, transmission_dla = insert_dlas(trans_wave, metadata['Z'][ii], rstate=random_state_just_for_dlas)
+                for idla in dlas:
+                   idla['dlaid']+=idd*1000      #Added to have unique DLA ids. Same format as DLAs from file. 
```

Comment:
`DLAID` is already unique I think. Why add `idd*1000`?

Reply:
@londumas This is only for the case when we add DLAs from random, not from transmission file. This is to match the same format as it is now in the case of adding DLAS from transmission file.